src/Chapter1.hs

```diff
@@ -568,7 +576,14 @@ True
 >>> isVowel 'x'
 False
 -}
-isVowel c = error "isVowel: not implemented!"
+isVowel :: Char -> Bool
+isVowel c
+  | c == 'a' || c == 'A' = True 
+  | c == 'e' || c == 'E' = True
+  | c == 'i' || c == 'I' = True
+  | c == 'o' || c == 'O' = True 
+  | c == 'u' || c == 'U' = True
```

Comment:
Awesome solution! Absolutely correct!  👍🏻 

As `||` alread returns a value of type `Bool`, you don't need to use guards and return the result of operator application directly:

```haskell
isVowel c =
        c == 'a' || c == 'A'
    ||  c == 'e' || c == 'E'
    || ...
```